pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[build-system]
+requires = [
+    "setuptools >= 35.0.2",
+    "wheel >= 0.29.0",
+    "setuptools_scm"
+]
```

Comment:
Where did these specific versions for `setuptools` and `wheel` come from, and why use version specifiers for those two but not for `setuptools_scm`? Can we just take out the version specifiers for those two, assuming that any version that is new enough to be pyproject.toml-aware is compatible?

Reply:
should be the minimum needed, choose some arbitrary. Ideally should be something we can test with.

Reply:
Wait so those are the exact minimum versions needed or they were chosen arbitrarily? In 1f5a66c355d5b43dfa8bbb4c7f78cbf106e0f98b I went with bare package names rather than choosing arbitrary versions. But I'd be happy to change to compatibility-preserving version ranges like `>= X.Y, < (X+1).Y` as long as the X.Y is actually the true minimum required version and not something arbitrary.

Reply:
Arbitrarly choosen version that for sure should contain all you're using inside setup.py.